### PR TITLE
Endzyme's PR w/ my changes

### PR DESCRIPTION
**Pull request by blimmer**
cc @endzyme

This is based on your original PR #1 - lmk what you think

### CODE REVIEW

File: zsh-aws-vault.plugin.zsh [hunk 1]
```diff
@@ -11,6 +11,7 @@ alias av='aws-vault'
 alias ave='aws-vault exec'
 alias avl='aws-vault login'
 alias avs='aws-vault server'
+alias avll='avl -s'
```
Comment (blimmer) [inline zsh-aws-vault.plugin.zsh:14]: I thought this was value to expose to end users

File: zsh-aws-vault.plugin.zsh [hunk 2]
```diff
@@ -19,6 +20,48 @@ function avsh() {
...
+        ;;
+    esac
+
+  else
+    # NOTE this is untested - PRs welcome to improve it.
+    echo "${login_url}" | xargs xdg-open
+  fi
+}
+
+function avp() {
```
Comment (blimmer) [inline zsh-aws-vault.plugin.zsh:48]: this is great - I just made the naming match up a bit better with the existing values

File: zsh-aws-vault.plugin.zsh [hunk 3]
```diff
@@ -31,3 +74,22 @@ function prompt_aws_vault_segment() {
...
+#--------------------------------------------------------------------#
+# Utility Functions                                                  #
+#--------------------------------------------------------------------#
+function _using_osx() {
+  [[ $(uname) == "Darwin" ]]
+}
+
+function _find_browser() {
+  if _using_osx ; then
+    # https://stackoverflow.com/a/32465364/808678
```
Comment (blimmer) [inline zsh-aws-vault.plugin.zsh:87]: This works pretty well!

**Comment by endzyme** [comment]
Looks great. Thanks @blimmer

**Pull request merged**